src/commands/ImportCommand.js

```diff
@@ -0,0 +1,76 @@
+import fs from "fs";
+import path from "path";
+import child from "child_process";
+import Command from "../Command";
+import progressBar from "../progressBar";
+import PromptUtilities from "../PromptUtilities";
+
+export default class ImportCommand extends Command {
+  initialize(callback) {
+    const inputPath = this.input[0];
+
+    if (!inputPath) {
+      callback(new Error("Missing argument: Path to foreign repository"));
+    }
+
+    const foreignRepoPath = path.resolve(inputPath);
+
+    try {
+      const stats = fs.statSync(foreignRepoPath);
+      if (!stats.isDirectory()) {
+        throw new Error(`Input path "${inputPath}" is not a directory`);
+      }
+      const packageJson = path.join(foreignRepoPath, "package.json");
+      const packageName = require(packageJson).name;
+      if (!packageName) {
+        throw new Error(`No package name specified in "${packageJson}"`);
+      }
+      this.targetDir = "packages/"+packageName;
+    } catch (e) {
+      callback(e);
+    }
+
+    this.foreignExecOpts = {
```

Comment:
`externalExecSync` is also called from the `execute` method, so the options need to persist until then.

Could, alternatively, attach only `externalRepoPath` to `this` and build up the exec options inside `externalExecSync`, if you'd prefer.